assets/launcher/lang/sklmessages_cs_CZ.properties

```diff
@@ -55,52 +55,58 @@ editor.name=Název Profilu
 editor.copy=Kopie %s
 editor.gamedir=Místo uložení hry
 editor.resolution=Rozlišení
-editor.visibility=Viditelnost Launcheru
-editor.hide=Schovat launcher a znovu otevřít po ukončení hry
-editor.close=Ukončit launcher když se nastartuje hra
-editor.keep=Nechat launcher otevřený
-editor.snap=Snapshoty
-editor.snap.tooltip=Povolit použití experimentálních vývojových verzí ("snapshoty")
+editor.visibility=Viditelnost spouštěče
+editor.hide=Schovat spouštěč a znovu otevřít po ukončení hry
+editor.close=Ukončit spouštěč když se nastartuje hra
+editor.keep=Nechat spouštěč otevřený
```

Comment:
Is it necessary to translate the word "launcher"? In my opinion, not translating it would be the better solution. It also refers to the name "SKlauncher".